agent/Makefile

```diff
@@ -1,25 +1,32 @@
 CC=gcc
 DISTRO:=$(shell lsb_release -is)
-override CFLAGS:=-I. -I../libqrexec -g -O2 -Wall -Wextra -Werror `pkg-config --cflags vchan-$(BACKEND_VMM)`$(CFLAGS)
-# LDFLAGS=-pie
-LDLIBS=`pkg-config --libs vchan-$(BACKEND_VMM)` -lqrexec-utils -lpam
+override QUBES_CFLAGS := -I. -I../libqrexec -g -O2 -Wall -Wextra -Werror \
+   `pkg-config --cflags vchan-$(BACKEND_VMM)` -fstack-protector \
+   -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fPIC \
+   $(CFLAGS)
+override LDFLAGS += -pie -Wl,-z,relro,-z,now
+override LDLIBS=`pkg-config --libs vchan-$(BACKEND_VMM)` -lqrexec-utils -lpam
+
+.PHONY: all install
 
 all: qrexec-agent qrexec-client-vm qrexec-fork-server #qrexec-client-vm.1.gz
 qrexec-agent: qrexec-agent.o qrexec-agent-data.o
 qrexec-fork-server: qrexec-fork-server.o qrexec-agent-data.o
 qrexec-client-vm: qrexec-client-vm.o qrexec-agent-data.o
 clean:
-	rm -f -- *.o *~ qrexec-agent qrexec-client-vm
+	rm -f -- *.o *~ qrexec-agent qrexec-client-vm *.o.dep
 
-install:
-	install -d $(DESTDIR)/etc/qubes-rpc
-	install -d $(DESTDIR)/usr/lib/qubes $(DESTDIR)/usr/bin
+install: all
+	install -d $(DESTDIR)/etc/qubes-rpc $(DESTDIR)/usr/lib/qubes \
+		$(DESTDIR)/usr/bin $(DESTDIR)/usr/share/main/man1
 	install qrexec-agent $(DESTDIR)/usr/lib/qubes
 	install qrexec-client-vm $(DESTDIR)/usr/bin
-	install qrexec-client-vm.1.gz $(DESTDIR)/usr/share/man/man1 || :
+	if [ -f qrexec-client-vm.1.gz ]; then \
+		install qrexec-client-vm.1.gz $(DESTDIR)/usr/share/man/man1; \
+	fi
```

Comment:
I'd rather prefer the build fail if the file is missing. I've added the missing file to the master branch. Other problem is lack of `PANDOC=pandoc` setting, but I haven't added it there, to not cause conflicts.